src/notte/errors/handler.py

```diff
@@ -46,14 +47,22 @@ async def wrapper(*args: Any, **kwargs: Any) -> T:
             logging.error(f"NotteBaseError: {e.dev_message}")
             raise e
         except PlaywrightTimeoutError as e:
-            # logging.error(f"PlaywrightTimeoutError with message: {e.message}")
-            # Transform external timeout error
-            if "- waiting for locator(" in str(e):
+            # only timeout issue if the last line is it
+            # otherwise more generic error
+            if "- waiting for locator(" in str(e).strip().split("\n")[-1]:
                 raise InvalidLocatorRuntimeError(message=str(e)) from e
+
             raise PlaywrightRuntimeError(message=str(e)) from e
         except TimeoutError as e:
             raise NotteTimeoutError(message="Request timed out.") from e
         # Add more except blocks for other external errors
+        except PlayrightError as e:
+            raise NotteBaseError(
+                dev_message=f"Unexpected playwright error: {str(e)}",
+                user_message="An unexpected playwright error occurred. Our team has been notified.",
```

Comment:
no need to mention playright in `user_message` (these are for the users of the api that may not have any idea about what playwright is)